lib/ansible/modules/network/cli/cli_config.py

```diff
@@ -290,6 +290,7 @@ def run(module, device_operations, connection, candidate, running, rollback_id):
             if commit:
                 connection.edit_config(**kwargs)
             result['changed'] = True
+            result['commands'] = (str(config_diff).split('\n'))    
```

Comment:
With this, the `commands` key won't be returned for platforms that support on box diff. This needs to be handled in `supports_onbox_diff` block too. 

However, on box diff, usually doesn't return commands, instead, it returns something like:
```
edit interfaces ethernet eth1]
+description "Configured by Ansible - Interface 1"
+mtu 1500
+vif 100 {
+    description "Eth1 - VIF 100"
+    mtu 400
+}
+vif 101 {
+    description "Eth1 - VIF 101"
+}
[edit interfaces]
+ethernet eth2 {
+    description "Configured by Ansible - Interface 2 (ADMIN DOWN)"
+    disable
+    duplex auto
+    mtu 600
+    smp_affinity auto
+    speed auto
+}
[edit]
```